app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java

```diff
@@ -171,13 +171,18 @@ public void askReasonAndExecute(Media media,
             reasonListEnglish[1] = getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_spam_blurry);
             reasonListEnglish[2] = getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_spam_nonsense);
         } else if (problem == ReviewController.DeleteReason.COPYRIGHT_VIOLATION) {
+            reasonList = new String[]{"Reason 1", "Reason 2", "Reason 3", "Reason 4"};
+            reasonListEnglish = new String[]{"Eng1", "Eng2", "Eng3", "Eng4"};
             reasonList[0] = context.getString(R.string.delete_helper_ask_reason_copyright_press_photo);
             reasonList[1] = context.getString(R.string.delete_helper_ask_reason_copyright_internet_photo);
             reasonList[2] = context.getString(R.string.delete_helper_ask_reason_copyright_logo);
+            reasonList[3] = context.getString(R.string.delete_helper_ask_reason_copyright_panorama);
             reasonListEnglish[0] = getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_reason_copyright_press_photo);
             reasonListEnglish[1] = getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_reason_copyright_internet_photo);
             reasonListEnglish[2] = getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_reason_copyright_logo);
+            reasonListEnglish[3] = getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_reason_copyright_panorama);
         }
+        final String[] newReasonListEnglish = reasonListEnglish;
```

Comment:
What is the goal of this line change? The goal is not to solve #4797 right, is it another issue number?

Reply:
it`s because I previously changed reasonListEnglish to non-final so that it can be modified, so I added this here and replaced its original use with newReasonListEnglish which is final so the original purposes would not be affected.

Reply:
I think it has to be final here, when I remove this line and rerun the app on my end it would show this error
![pr image](https://user-images.githubusercontent.com/111096054/199175617-b25865a3-86dd-4345-9be3-80d21323fdae.jpg)
This part wasn`t affected before since the array reasonListEnglish was final before the pull request, but since I changed it, I added this newReasonListEnglish which is final to replace the original uses of reasonListEnglish.

Reply:
Thanks for explaining you reasoning!

> reasonListEnglish was final before the pull request

In master it does not seem to be final though: https://github.com/commons-app/apps-android-commons/blob/c626f97fb95621dd0846d8c5fb06c5bd8fac7bdd/app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java#L164
You are able to build the master branch, right?

Reply:
I`ve updated my fork to the newest version to make sure that I`m on the same page as the master branch, I think the problem here is that even though reasonListEnglish wasn`t a final variable before, it was used as an effectively final variable, so that it could be used in the lambda expression.
But in my pull request, I redefined and changed the length of reasonListEnglish and it wasn`t effectively final anymore, so I had to create newReasonListEnglish as to be used in the lambda expression. As it shows below, even when reasonListEnglish wasn`t originally final, when I made this change an error would show up.
![1](https://user-images.githubusercontent.com/111096054/199316510-6763d48c-2b73-438d-a890-fb8cd46aa9bb.jpg)
The error disappears after I add newReasonListEnglish:
![2](https://user-images.githubusercontent.com/111096054/199316883-843ef498-2195-4eff-aa3d-a45900d37ae1.jpg)
So I think it`s still necessary to keep the newReaonListEnglish in here, as it would continue doing the functions of reasonListEnglish to work in line 204. 
I accidently closed this pull request when I was updating on my own fork, so I`ve created another one #5109, can you help me look at that one instead? It`s just the same as this current one. Thank you!